docker/jenkins/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+[ -d build ] && rm -rf build
+mkdir build && cd build
+cmake \
+  -D CMAKE_CXX_FLAGS="-Wall -Wextra" \
+  -D CMAKE_CXX_COMPILER=g++ \
+  -D PYTHON_EXECUTABLE=/usr/bin/python3 \
+  -D Tasmanian_ENABLE_MATLAB=OFF \
```

Comment:
I fixed that, MATLAB is disabled by default and this line is not needed any more.

Reply:
Will do.